.editorconfig

```diff
@@ -0,0 +1,13 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+root = true
+
+[*]
+end_of_line = crlf
```

Comment:
should this not be `lf`? or am i missing something?

Reply:
They had `crlf` before this config went in and I didn't want to change it _for all files_ this time, because it will be confusing with the other changes

I've explicitly set shell files to `lf` because (although we don't have many, and most of them are developer-only) they have been exploding in Catalina which won't execute shell scripts with `crlf` endings

Reply:
What I might do, though, now the linting changes are made ... is flip that `*` line ending over to `lf` now, then everything will pick up the change as each file is edited normally ... hm!